### PR TITLE
Updates and Cleanup.

### PR DESCRIPTION
**Pull request by Kkthnx**
Some cleanup and updates and quality of life changes.

wowscrnshot_102716_184416

### CODE REVIEW

**Comment by Kkthnx** [comment]
The nameplates could really use a major cleanup. I think the biggest thing right now is getting the nameplates in place and get the personal resource bar to display buffs and debuff as it was intended to use. Or just completely move over to spawning them with oUF?

File: ShestakUI/Modules/Blizzard/Nameplates.lua [hunk 1]
```diff
@@ -873,8 +872,7 @@ end
 
 local function CreateAuraIcon(parent)
 	local button = CreateFrame("Frame", nil, parent)
-	button:SetWidth(C.nameplate.auras_size)
-	button:SetHeight(C.nameplate.auras_size)
+	button:SetSize(C.nameplate.auras_size, C.nameplate.auras_size * 16/25)
```
Comment (Shestak) [inline ShestakUI/Modules/Blizzard/Nameplates.lua]: Why "* 16/25" for one size?

**Comment by Wetxius** [comment]
Please, create separate pull requests for each change.

**Comment by Wetxius** [comment]
I rewrite nameplate twice in this year, really not even a hint of inspiration to rework their for oUF.

**Comment by Kkthnx** [comment]
Understood.

**Comment by Kkthnx** [comment]
If you want just cherrypick these changes I won't be able to make new requests for about 6 hours.

**Comment by Wetxius** [comment]
http://shestak.org/forum/showthread.php?t=354

**Comment by Wetxius** [comment]
I can implement showing all debuffs from Blizzard list, by checking boolean nameplateShowAll 
local dname, _, _, _, _, dduration, _, dcaster, _, _, dspellid, _, _, _, nameplateShowAll = UnitAura(unit, index, 'HARMFUL')

**Comment by Kkthnx** [comment]
I personally think we should filter it like we do already for debuffs. I can build a list really quick for us to use. Though it is, however, you wanna do this. If this is planned already I can give a list.

**Comment by Wetxius** [comment]
Blizzard shows only important debuffs, isn't it?

**Comment by Kkthnx** [comment]
I do believe so. But filtering it is always a good choice? Meh, it is totally up to you.

**Comment by Wetxius** [comment]
We can shows Blizzard list and added out whitelist and blacklics list

**Comment by Kkthnx** [comment]
Okay sounds good.

**Pull request closed without merging**